.github/workflows/lint.yml

```diff
@@ -0,0 +1,31 @@
+name: lint
+on:
+  pull_request:
+    paths:
+      - 'console/**'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    env:
+      working-directory: ./console
+
+    steps:
+      - uses: actions/checkout@v2
+
```

Comment:
I am not very sure, Will this cache the npm libraries if the lock file has not changed?

What do you think about a cache here? 
https://github.com/actions/cache/blob/master/examples.md#node---npm

https://help.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows
@beerose

Reply:
It's okay to go without cache for now.